app/models/services/base-service.ts

```diff
@@ -544,6 +544,9 @@ export default abstract class BaseService<ServiceParamType> {
     if (operation.requireSynchronous) {
```

Comment:
This is interesting -- I just found out that WMS requests must be synchronous. I failed to find a description of WMS request in Harmony documentation. We should consider put this in the future Harmony landing documentation page.